integration_test/integration_tests.py

```diff
@@ -23,7 +23,7 @@ def validate(json_to_validate, schema):
     validator.validate(json_to_validate, schema)
 
 
-def send_sms_notification_test_response(python_client, sender_id=None):
+def send_sms_notification_test_response(python_client, sender_id=None, callbackUrl=None, bearerToken=None):
```

Comment:
`callbackUrl` => `status_callback_url` and `bearerToken` => `status_callback_bearer_token`